mlflow/shap.py

```diff
@@ -1,20 +1,90 @@
 from contextlib import contextmanager
 import os
 import tempfile
+import pickle
+import yaml
+import shap
```

Comment:
This import needs to be moved into the associated functions to avoid early imports of optional packages

Reply:
moved imports into function